### PR TITLE
test: address regression pipeline slow down

### PR DESCRIPTION
**Pull request by cortisiko**
## **Description**

The purpose of this PR is to:
1. Add some critical flow accounts e2e tests to the smoke pipeline
2. Fix glob for tests tagged SmokeCore
3. Fix Portfolio dapp connect test. 
## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
4.
5.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -664,7 +664,7 @@ workflows:
       - ios_api_specs
   run_tag_smoke_core_ios:
     envs:
-      - TEST_SUITE_FOLDER: './e2e/specs/*/**/*'
+      - TEST_SUITE_FOLDER: './e2e/spec/*/**/*'
```
Comment (chrisleewilcox) [inline bitrise.yml:667]: Is this change intended?  Are we renaming the folder?

File: bitrise.yml [hunk 2]
```diff
@@ -674,7 +674,7 @@ workflows:
         stack: linux-docker-android-22.04
         machine_type_id: elite-xl
     envs:
-      - TEST_SUITE_FOLDER: './e2e/specs/*/**/*'
+      - TEST_SUITE_FOLDER: './e2e/spec/*/**/*'
```
Comment (chrisleewilcox) [inline bitrise.yml:677]: Same as first comment.

File: bitrise.yml (hunk 1 above)
Comment (cortisiko) [reply bitrise.yml:667]: This was a changed made by me in one of my previous PRs which breaks the smoke core workflows. I am reverting that change to what it was previously. https://github.com/MetaMask/metamask-mobile/blob/5ed686b2d4f440a8a20293f1a0af76f2922cf27c/bitrise.yml#L647

**Review by tommasini — APPROVED** [review]
